dxr/plugins/clang/condense.py

```diff
@@ -57,6 +57,14 @@ def process_function(props):
     props['type'] = c_type_sig(input_args, props['type'])
     return props
 
+def process_maybe_override(overrides, overriddens, props):
+    qualname = props.get('qualname')
+    if qualname in overrides or qualname in overriddens:
+        # Mark this function as virtual.
```

Comment:
I don't think a function's presence in `overrides` or `overriddens` implies that it's virtual. After all, you can override a nonvirtual function; dispatch is just not as fancy.